lib/loader.js

```diff
@@ -82,8 +82,13 @@ module.exports = function(content, map) {
 				"[" + JSON.stringify(importItem.export) + "] + \"";
 		}
 
+		var escapeUrlHelper;
```

Comment:
```js
// url() helper to escape quotes
var escape = ""
```

`\n` after the variable decl please

Reply:
The variable is now named `urlEscapeHelper` in the loader and `escape` in the output. If you feel that is incorrect could you be a bit more specific about the naming you would prefer?